README.md

```diff
@@ -10,8 +10,10 @@ cargo install stellar-contract-cli --git https://github.com/stellar/stellar-cont
 
 ## Usage
 
+### Directly invoking a function in auser specified WASM contract
+
 ```
-stellar-contract-cli invoke --file <WASMFILE> --fn <FUNCNAME> --arg i32:1 --arg i32:2
+stellar-contract-cli invoke --file <WASMFILE> vm-fn --fn <FUNCNAME> --arg i32:1 --arg i32:2
```

Comment:
It's not obvious that `vm-fn` means a contract fn for a user uploaded contract. I think we should keep the top-level subcommand `invoke` as to mean invoking a contract fn. We should try to abstract some of the oddities of the fact that all fns are host called, the CLI doesn't need to expose this.

Reply:
I removed the subcommands I added.